infrastructure/ansible/roles/traffic_ops/templates/cdn.conf.j2

```diff
@@ -81,7 +81,9 @@
          "password": "{{ to_riak_username_password }}",
          "MaxTLSVersion": "{{ to_riak_tls_max_version }}"
 {% endif %}
-      }
+      },
+      "whitelisted_oauth_urls": {{ to_whitelisted_oauth_urls | to_json }},
+      "oauth_client_secret": "{{ to_oauth_client_secret }}"
```

Comment:
`to_oauth_client_secret` is not in `roles/traffic_ops/defaults/main.yml`. It should be added in the `to_go_` section:

https://github.com/apache/trafficcontrol/blob/71c405c8f56ccbdc848ddab962bd2c9f9a65df79/infrastructure/ansible/roles/traffic_ops/defaults/main.yml#L75-L77

Reply:
Done